tests/NightModeUserSettingModal.test.ts

```
@@ -0,0 +1,131 @@
+      return true;
+    }
+    createRenderRoot() {
+      return this;
+    }
+  },
+  html: (strings: TemplateStringsArray, ...values: any[]) =>
+    String.raw(strings, ...values),
+}));
+
+jest.mock("lit/decorators.js", () => ({
+  customElement: (name: string) => (constructor: any) => constructor,
```

Biome (2.1.2)

[error] Do not shadow the global "constructor" property.

Consider renaming this variable. It's easy to confuse the origin of variables when they're named after a known global.

(lint/suspicious/noShadowRestrictedNames)